taca/nanopore/ONT_run_classes.py

```diff
@@ -338,6 +350,113 @@ def copy_html_report(self):
             logger.error(msg)
             raise RsyncError(msg)
 
+    def toulligqc_report(self):
+        """Generate a QC report for the run using ToulligQC and publish it to GenStat."""
+
+        # Get sequencing summary file
+        glob_summary = glob.glob(f"{self.run_abspath}/sequencing_summary*.txt")
+        assert len(glob_summary) == 1, f"Found {len(glob_summary)} summary files"
+        summary = glob_summary[0]
+
+        # Determine the format of the raw sequencing data, sorted by preference
+        raw_data_dir_options = [
+            "pod5_pass",
+            "pod5",
+            "fast5_pass",
+            "fast5",
+        ]
+        raw_data_dir = None
+        for raw_data_dir_option in raw_data_dir_options:
+            if os.path.exists(f"{self.run_abspath}/{raw_data_dir_option}"):
+                raw_data_dir = raw_data_dir_option
+                break
+        if raw_data_dir is None:
+            raise AssertionError(f"No seq data found in {self.run_abspath}")
+        raw_data_format = "pod5" if "pod5" in raw_data_dir else "fast5"
+
+        # Load samplesheet, if any
+        ss_glob = glob.glob(f"{self.run_abspath}/sample_sheet*.csv")
+        if len(ss_glob) == 0:
+            samplesheet = None
+        elif len(ss_glob) > 1:
+            # If multiple samplesheet, use latest one
+            samplesheet = ss_glob.sort()[-1]
```

Comment:
Sort would sort them on file name? And the samplesheet file name starts with a timestamp?

Reply:
Yes, they are named w. the following
```
file_name = f"MinKNOW_samplesheet_{process.id}_{TIMESTAMP}_{process.technician.name.replace(' ','')}.csv"
```

Reply:
`process.id` is a LIMS process? That should probably work, even if I'm not entirely comfortable in relying on LIMS internal convention to sort the files.